recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+# TODO: Once https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/688
+#       is resolved, remove this custom graphviz pinning.
+graphviz:  # [win]
+  - 2.38   # [win]
```

Comment:
I'm no expert on Graphviz at all. But this should be fine given:
- the [`max_pin: x` in `conda-forge-pinning`](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/cc418d28617f0581baf74a7f0feb73c7f2c57844/recipe/conda_build_config.yaml#L188-L189)
- and the track record from https://abi-laboratory.pro/?view=timeline&l=graphviz .

Reply:
Yeah this is the right thing to do I think. We are just doing binary repackaging on Windows anyways (due to the complexity of the Graphviz build system). Also `2.38.0` was the last binary they uploaded for Windows. Maybe this will be easier to build from source with CMake support they added ( https://github.com/conda-forge/graphviz-feedstock/issues/11 ). All that to say this seems like a good solution for now. 🙂